src/ioctls/vm.rs

```diff
@@ -192,6 +192,45 @@ impl VmFd {
         }
     }
 
+    /// Directly injects a MSI message as per the `KVM_SIGNAL_MSI` ioctl.
+    ///
+    /// # Arguments
+    ///
+    /// * kvm_msi - MSI message configuration. For details check the `kvm_msi` structure in the
+    ///                [KVM API doc](https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt).
+    /// # Example
+    ///
+    /// ```rust
+    /// # extern crate kvm_ioctls;
+    /// extern crate kvm_bindings;
+    /// # use kvm_ioctls::{Kvm, VmFd};
+    /// use kvm_bindings::kvm_msi;
+    ///
+    /// let kvm = Kvm::new().unwrap();
+    /// let vm = kvm.create_vm().unwrap();
+    /// let msi = kvm_msi::default();
+    /// #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    /// vm.create_irq_chip().unwrap();
+    /// //vm.signal_msi(msi).unwrap();
```

Comment:
Maybe add a comment here to why the signal_msi is commented out?

Reply:
Yes will do.

Reply:
done